config/prow/config_knative.yaml

```diff
@@ -433,6 +433,39 @@ periodics:
           memory: 12Gi
         limits:
           memory: 16Gi
+    - dot-release: true
+      dot-dev: true
+      go113: true
+      release: "0.10"
+      env-vars:
+      - RELEASE_BRANCH=releaes-0.10
```

Comment:
typo - `releaes` -> `release`, the same for all other configs.

Reply:
done